src/package.h

```diff
@@ -0,0 +1,38 @@
+#ifndef CYCLUS_SRC_PACKAGE_H_
+#define CYCLUS_SRC_PACKAGE_H_
+
+#include <string>
+#include <boost/shared_ptr.hpp>
```

Comment:
This should fix your build error

```suggestion
#include <limits>
#include <string>
#include <boost/shared_ptr.hpp>
```